### PR TITLE
[Merged by Bors] - feat(Data/Int/WithZero): add API

### PR DESCRIPTION
**Pull request by fbarroero**
In this PR we add three basic facts about `WithZeroMultInt.toNNReal`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Int/WithZero.lean [hunk 1]
```diff
@@ -88,4 +88,38 @@ theorem toNNReal_strictMono {e : NNReal} (he : 1 < e) :
...
+  · exact fun h ↦ mono h
+
+theorem toNNReal_le_one_iff {e : NNReal} {m : ℤₘ₀} (he : 1 < e) :
+    toNNReal (ne_zero_of_lt he) m ≤ 1 ↔ m ≤ 1 := by
+  have mono := toNNReal_strictMono he
+  constructor
+  · apply le_imp_le_of_lt_imp_lt
+    exact fun a ↦ LT.lt.trans_eq' (mono a) rfl
+  · apply le_imp_le_of_lt_imp_lt
+    exact fun h ↦ (StrictMono.lt_iff_lt mono).mp h
```
Comment (mariainesdff) [inline Mathlib/Data/Int/WithZero.lean:123]:
```suggestion
  have : 1 = (toNNReal (ne_zero_of_lt he)) 1 := rfl
  simp_rw [this]
  exact StrictMono.le_iff_le (toNNReal_strictMono he)
```

File: Mathlib/Data/Int/WithZero.lean [hunk 2]
```diff
@@ -88,4 +88,38 @@ theorem toNNReal_strictMono {e : NNReal} (he : 1 < e) :
       WithZero.coe_unzero hy]
     exact hxy
 
+theorem toNNReal_eq_one_iff {e : NNReal} (m : ℤₘ₀) (he0 : e ≠ 0) (he1 : e ≠ 1) :
```
Comment (mariainesdff) [inline Mathlib/Data/Int/WithZero.lean:91]: Would you mind replacing all of the `{e : NNReal} ` in the file by `{e : ℝ≥0}`? (I know that not all of these were introduced by this PR, so I will leave it up to you).

Comment (fbarroero) [reply Mathlib/Data/Int/WithZero.lean]: Done!

**Comment by fpvandoorn** [comment]
bors merge

**Pull request closed without merging**